protocol/v2/ssv/validator/events.go

```diff
@@ -42,13 +43,22 @@ func (c *Committee) handleEventMessage(logger *zap.Logger, msg *queue.DecodedSSV
 		if err != nil {
 			return err
 		}
+
 		c.mtx.Lock()
-		dutyRunner := c.Runners[slot] // TODO: err check , runner exist?
+		q := c.Queues[slot]
+		dutyRunner, exists := c.Runners[slot]
 		c.mtx.Unlock()
+		if !exists {
```

Comment:
```suggestion
		q, exists := c.Queues[slot]
		dutyRunner, exists := c.Runners[slot]
		c.mtx.Unlock()
		if !exists || !qexists {
```

queue should be present if there's a runner but lets be on the safe side anyway.